### DISCUSSION TITLE
Set wrong address in Target EID TLV

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#9942] [Q&A] [answered]
Hello,

Test case 5.9.20 DUA fails in most trials for router role from thread harness because of wrong address in Target EID TLV.

it seems like DUA is not added to DUT!

Attached logs, capture and report for failure.
[5.9.20.zip](https://github.com/openthread/openthread/files/14635500/5.9.20.zip)

DUT is using latest release from openthread with configuration OT_THREAD_VERSION_1_3.
GRL v59 is used.

Is there any suspected reason that leads to use wrong address in target EID TLV ?

Thanks in advance

**Comment by jwhui** [comment, marked as answer]
This appears to be a Test Harness issue.

Packet 143 shows that the Domain Prefix configured is `fd00:7d03:7d03:7d03::/64`, which is consistent with what the DUT is including in the Target EID in Packet 144.
